config/src/trusted_peers.rs

```diff
@@ -81,6 +81,7 @@ pub struct ConsensusPeersConfig {
 pub struct UpstreamPeersConfig {
     /// List of PeerIds serialized as string.
     pub upstream_peers: Vec<String>,
+    // invariant self.upstream_peers.all(|peer_id_str| { PeerId::from_str(peer_id_str) })
```

Comment:
did you mean to have that here?

Reply:
It is documentation that reminds people that users of this field expect all of the strings to be parsable as valid PeerId instances.